README.md

```diff
@@ -79,19 +79,35 @@ Once completed, run:
 
 ### Database clear down between tests
 
-Firstly, whitelist you IP using the ras commands for the target environment. Then get the database credentials
+**NOTE**: The below commands **should** only ever be ran in dev (under your namespace) or in performance.
 
-`export [TARGET-NAMESPACE]_PASSWORD_[TARGET-SERVICE]=$(kubectl get secret [DB-CREDENTIALS] -o json --namespace=[TARGET-NAMESPACE] | jq -r '.data."[PASSWORD-KEY]"' | base64 -d)`
+Firstly, whitelist your IP using the ras commands for the dev or performance. Then get the database for either dev or 
+performance credentials:
 
-The above command will need to be replicated for all the schemas affected by the Locust tests. When using dev, this 
-will only be a single command.
+**DEV**
 
-Once the passwords have been exported to your local machine, run the following:
+`export DEV_PASSWORD_DATABASE=$(kubectl get secret [DB-CREDENTIALS] -o json --namespace=[YOUR-NAMESPACE] | jq -r '.data."[PASSWORD-KEY]"' | base64 -d)`
```

Comment:
This isn't needed for dev. It runs in a postgres container and there's one `ras` database with `u:postgres` and `p:postgres`

Reply:
Removed